Medium/scraping.py

```diff
@@ -2,22 +2,22 @@
 import requests
 
 
-def tag_search(tag):
+def tag_search_scrape(tag):
     source = requests.get(f"https://www.medium.com/tag/{tag}").text
     soup = BeautifulSoup(source, "lxml")
-    link = []
+    links = []
     for part in soup.findAll("div", class_="postArticle-readMore"):
-        link.append(part.a["href"])
-    return link
+        links.append(part.a["href"])
+    return links
 
 
-def search(search, num):
+def search_scrape(search):
     source = requests.get(f"https://www.medium.com/search?q={search}").text
     soup = BeautifulSoup(source, "lxml")
     i = 0
     link = []
     for part in soup.findAll("div", class_="postArticle-readMore"):
-        if i == num:  # Number of articles that need to be fetched
+        if i == 3:  # Number of articles that need to be fetched
```

Comment:
Why 3? I assume we want it to be dynamic in nature? :thinking:

Reply:
That is just a placeholder for now, I have planned a lot more. I have divided the project of the Medium Bot into some more phases, so this issue will be resolved soon!